src/ontology/edits/oeo-physical.omn

```diff
@@ -810,7 +810,7 @@ Class: CarbonDioxideEmission
 Class: Charcoal
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "The solid residue of the destructive distillation and pyrolysis of wood and other vegetal material."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Charcoal is a portion of matter consisting of the solid residue of the destructive distillation and pyrolysis of wood and other vegetal material."@en,
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Charcoal is a fuel consisting of the solid residue of the destructive distillation and pyrolysis of wood and other vegetal material."@en,
```

Reply:
Charcoal is primarily a subclass of portion of matter. Would it be enough to add another sentence like "It is used as a fuel" at the end?